specification/04-requirements.adoc

```diff
@@ -167,22 +167,29 @@ Management (`IDE_KM`) support, and its ability to receive IDE_KM messages over a
 Secured SPDM session. However, there are no architecturally-defined PCIe
 protocol for managing Root Port IDE keys.
 
-Instead of adding multiple vendor-specific `IDE_KM` implementations to the TSM,
-the TSM relies on the platform Root-of-Trust (ROT) to implement the `IDE_KM`
-protocol and abstract the platform specific PCIe RP implementation away from
-the TSM. The TSM establishes a Secured SPDM session with the ROT over a host
-accessible DOE mailbox, and then sets platform RP IDE keys over that session.
+When the the platform Root-of-Trust (RoT) implements the `IDE_KM` protocol,
```

Comment:
Suggestion:  This paragraph starts with the SPDM/IDE variant, then mentions vendors specific solutions, and finally returns to the SPDM/IDE with corresponding requirements.
Could we make the distinction more explicit, by using the following intro:

"This specification considers two variants for conveying IDE keys from the TSM to the PCIe root port.
In one variant the platform provides vendor-specific protection for the communication path between these entities.
Here the TSM may program IDE keys directly through the `IDE_KM` protocol or choose any other vendor-specific mechanism.
In the second variant this protection is not provided by the platform and/or PCIe root ports are not directly accessible to the TSM."

The ordering here does not indicate any preference - I put the vendor-specific variant first, simply because there is not much to be stated and the rest can focus on the SPDM/IDE solution.

Reply:
Thanks for the suggestion @eckhard-delfs-qualcomm, it makes sense to me.

I restructured these paragraphs accordingly, let me know if that looks better to you.